### DISCUSSION TITLE
Localization issue

### DISCUSSION THREAD

**Discussion by MaxGasta** [OrchardCMS/OrchardCore#7217] [Q&A]
Hello,
I have two cultures IT and  EN,    IT is the Default culture.

I have some news  items.
I can't have news only in EN , always I have to create an IT version for first, and after change the culture.
In this way I get a fake IT item

When I try to change the culture  I get the message "Please save your content item before creating localizations"

Is there a way to have an item just in one "Not default" culture?

Thanks for help

**Comment by hishamco** [comment]
AFAIK from content localization module, that you should create your content in the default culture first, then you could localize it into the supported cultures

**Comment by sebastienros** [comment]
Looks like a bad limitation. You should file an issue, and we should provide a way to do it.
